Makefile

```diff
@@ -167,7 +167,7 @@ ready: manifests generate fmt fmt_license go.mod vet lint test ## Make the code
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
+	GOMEGA_DEFAULT_EVENTUALLY_TIMEOUT=10s KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out -covermode=atomic -coverpkg=./...
```

Comment:
can you explain what `-covermode=atomic` does?

Reply:
This is just copied from SPI repo. Looking at the docs, it probably isn't needed but would be if we ran the tests in concurrently.